Push/PushClient.swift

```diff
@@ -3,7 +3,6 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import Foundation
```

Comment:
This file can be removed, rebase to latest development to see that it;s gone now